administrator/components/com_languages/access.xml

```diff
@@ -8,4 +8,12 @@
 		<action name="core.edit" title="JACTION_EDIT" description="JACTION_EDIT_COMPONENT_DESC" />
 		<action name="core.edit.state" title="JACTION_EDITSTATE" description="JACTION_EDITSTATE_COMPONENT_DESC" />
```

Comment:
don't we need the core.permission in the component too? to be the default (true) and fallback.